pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/Dispatcher.java

```diff
@@ -40,8 +40,18 @@
 
     boolean canUnsubscribe(Consumer consumer);
 
+    /**
+     * disconnect all consumers and mark dispatcher closed to stop new incoming requests
+     * 
+     * @return
+     */
     CompletableFuture<Void> disconnect();
```

Comment:
Perhaps we should also rename this method to something like `disconnectAllConsumers()`